processors/firehose_to_es_processor/lib/util.py

```diff
@@ -0,0 +1,54 @@
+import json
+
+
+def extract_json(message):
+        """
+            Returns extracted JSON if valid or None if not found or invalid from data string
+
+            Args:
+            data (string): Input log event data
+
+            Output:
+            dict
+        """
+        json_bound_loci = extract_json_bounds(message)
+        beginning_index = json_bound_loci[0]
+        end_index = json_bound_loci[1]
+        if beginning_index is not None and end_index is not None:
+            try:
+                formatted_data = message[beginning_index:end_index + 1]
+                json_valid_data = formatted_data.replace("'", '"')
```

Comment:
This will break valid json such as `{“a”: “Hi my name is ‘matt.’ How are you today?”}`. I would drop support of stringified python dictionaries. The tests that count on this functionality would also have to be modified.